scripts/zones/Beadeaux/Zone.lua

```diff
@@ -16,13 +16,20 @@ require("scripts/globals/zone")
 local zone_object = {}
 
 zone_object.onInitialize = function(zone)
-    -- Regions 1-6 are for the Afflictor System (RegionID, X, Radius, Z)
-    zone:registerRegion(1, -163, 10, -137, 0, 0, 0)
-    zone:registerRegion(2, -209, 10, -131, 0, 0, 0)
-    zone:registerRegion(3, -140, 10,   20, 0, 0, 0)
-    zone:registerRegion(4,  261, 10,  140, 0, 0, 0)
-    zone:registerRegion(5,  340, 10,  100, 0, 0, 0)
-    zone:registerRegion(6,  380, 10,   60, 0, 0, 0)
+    -- The Afflictor System (RegionID, X, Radius, Z) for curse
+    zone:registerRegion( 1, -163, 15, -137, 0, 0, 0) -- 17379801 The Afflictor
```

Comment:
Would suggest not using NPC ID, but some other method for locating it, since these are subject to change